exercises/I-array-properties/exercise.js

```diff
@@ -6,7 +6,10 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
+    if (arr.length === 0) {
+        return true;
+    }
+    return false;
```

Comment:
Good work - again, you could just return the result of the conditional statement.